packages/fx-core/resource/package.nls.json

```diff
@@ -23,6 +23,7 @@
   "core.deploy.noCapabilityFound": "No capability found to %s. Click Get Help to learn more about how to add capabilities.",
   "core.deploy.successNotice": "'%s' successfully deployed.",
   "core.deploy.aadManifestSuccessNotice": "Your Azure Active Directory application has been successfully deployed. Click \"Learn more\" to check how to view your Azure Active Directory application.",
+  "core.deploy.aadManifestOnCLISuccessNotice": "Your Azure Active Directory application has been successfully deployed.",
```

Comment:
Should we use word `update` instead of `deploy`? Because the CLI command is `teamsfx update aad-app`

Reply:
sure, thanks!